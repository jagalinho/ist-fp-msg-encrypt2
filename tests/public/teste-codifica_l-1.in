resul = codifica_l(faz_pos(2, 1), faz_pos(2, 4), 1)
resul
