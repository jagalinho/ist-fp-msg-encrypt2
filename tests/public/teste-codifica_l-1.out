(faz_pos(2, 2),  faz_pos(2, 0))
